### ISSUE TITLE
simulateProgress will still clearInterval  when loadingbar > 0

### ISSUE THREAD

**Issue by wangtao0101** [mironov/react-redux-loading-bar#32]
1 put(showLoading())
2 put(hideLoading())
3 put(showLoading())
4 put(hideLoading())
if 2 is very closely to 3, simulateProgress will ingnore the loadingbar state, and disappear the loading bar .

I solve the problem in this way:
```
componentWillReceiveProps(nextProps){
    if (this.shouldStart(nextProps)) {
        this.launch();
    } else if (this.shouldStop(nextProps)) {
        this.setState({ percent: this.props.maxProgress });
    }
}

simulateProgress() {
    let { progressInterval, percent, endingAnimationTimeout } = this.state
    const { maxProgress } = this.props

    if (percent === this.props.maxProgress && this.props.loading === 0) {
      clearInterval(progressInterval)
      endingAnimationTimeout = setTimeout(
        this.boundResetProgress,
        ANIMATION_TIME,
      )
      percent = 100
      progressInterval = null
    } else if (this.newPercent() <= maxProgress) {
      percent = this.newPercent()
    }

    this.setState({ percent, progressInterval, endingAnimationTimeout })
  }
```

**Comment by mironov** [comment]
@wangtao0101 I'm on it, thanks for reporting.

**Comment by mironov** [comment]
@wangtao0101 Published v2.7.3 with the fix.
